test/libs.js

```diff
@@ -53,8 +51,9 @@ const GENERATOR_PATH = path.join(__dirname, '../generators/app');
  * @return   {Array}
  *           An array of filenames.
  */
-const fileList = () =>
-  _.union.apply(_, _.toArray(arguments).map(str => FILES[str]));
+const fileList = function() {
+  return _.union.apply(_, _.toArray(arguments).map(str => FILES[str]));
```

Comment:
arguments was undefined for an arrow function, which turned this to an empty array, so it was searching for nothing...